src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAccount/Summary.cshtml

```diff
@@ -27,30 +27,30 @@
                     Address
                 </th>
                 <td colspan="2">
-                    @Model.Data.RegisteredAddress
+                    @foreach (var line in Model.Data.RegisteredAddress.Split(','))
```

Comment:
Could we move the split into the model so it can be tested?

Reply:
Moved into model